R/final_models.R

```diff
@@ -0,0 +1,651 @@
+#' Final Models
```

Comment:
Same comment as an earlier file. There may be repeated patterns that can be configured in a function.

Reply:
Discussed offline and decided to not fix based on limitations of foreach which can cause env export issues when running in parallel. 

https://github.com/RevolutionAnalytics/foreach/issues/34

Reply:
Discussed offline and decided to not fix based on limitations of foreach which can cause env export issues when running in parallel. 

https://github.com/RevolutionAnalytics/foreach/issues/34